espnet/nets/chainer_backend/ctc.py

```diff
@@ -127,6 +129,35 @@ def __call__(self, hs, ys):
 
         return self.loss
 
+    def forward_from_transformer(self, hs, ys):
```

Comment:
If this method is only for Transformer, I would like to find this in Transformer class